### PR TITLE
Fix lint not verifying all python source code #113

### PR DESCRIPTION
**Pull request by rmotitsuki**

### CODE REVIEW

File: tests/models/test_evc_base.py [hunk 1]
```diff
@@ -2,16 +2,14 @@
 import sys
 from unittest import TestCase
 
-from kytos.core.interface import UNI
+from napps.kytos.mef_eline.models import EVC
+from napps.kytos.mef_eline.scheduler import CircuitSchedule
+from tests.helpers import get_uni_mocked, get_controller_mock
 
 # pylint: disable=wrong-import-position
 sys.path.insert(0, '/var/lib/kytos/napps/..')
```
Comment (hdiogenes) [inline tests/models/test_evc_base.py]: @rmotitsuki, `sys.path.insert` must stay before the `napps` import, or it loses its function.

File: tests/models/test_link_protection.py [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
 from unittest.mock import Mock
 
 from kytos.core.common import EntityStatus
+from napps.kytos.mef_eline import settings
+from napps.kytos.mef_eline.models import EVC, Path  # NOQA
+from tests.helpers import MockResponse, get_link_mocked,\
+    get_uni_mocked, get_controller_mock  # NOQA
 
 # pylint: disable=wrong-import-position
 sys.path.insert(0, '/var/lib/kytos/napps/..')
```
Comment (hdiogenes) [inline tests/models/test_link_protection.py]: Same thing here.

File: tests/models/test_path.py [hunk 3]
```diff
@@ -43,18 +53,28 @@ def test_status_case_3(self):
...
+    @classmethod
+    def _mocked_requests_get_status_case_4(cls, url):
+        endpoint = '%s/%s' % (settings.TOPOLOGY_URL, 'links')
+        if url != endpoint:
+            raise ValueError('Not a topology URL')
+
+        return MockResponse({'links': [
+            {'active': True},
+            {'active': True}
+        ]}, 200)
```
Comment (hdiogenes) [inline tests/models/test_path.py]: I didn't understand this change. Is it really related to linting?

Comment (rmotitsuki) [reply tests/models/test_path.py]: I reverted the commit. It should be fixed in PR 119

File: tests/models/test_link_protection.py (hunk 2 above)
Comment (rmotitsuki) [reply tests/models/test_link_protection.py]: OK. fixed

File: tests/models/test_evc_base.py (hunk 1 above)
Comment (rmotitsuki) [reply tests/models/test_evc_base.py]: OK. fixed

**Pull request merged**